docs/development/audit-trail.md

```diff
@@ -0,0 +1,104 @@
+# Audit trail
+
+## Introduction
+The audit trail is a feature that allows you to track changes to your data. 
+It is useful for tracking changes to sensitive data, such as user accounts, or for tracking changes to data that is important for compliance, such as financial records.
+
+## Usage
+The audit trail is enabled by default for all entities.
```

Comment:
Het is niet mogelijk een volledige entity niet te loggen?

Reply:
Voor nu nog niet nee

Reply:
Niet per se een use case, was meer een gedachte